packages/flutter_tools/templates/app/lib/main.dart.tmpl

```diff
@@ -240,20 +240,20 @@ class _MyAppState extends State<MyApp> {
         ),
```

Comment:
We should re-format the templates too. `dart format` will change this. TODO self: file issue.

Reply:
Thanks for jumping on that! I agree.